Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/objc2/ObjectiveC2_Method.java

```diff
@@ -37,9 +37,13 @@ public ObjectiveC2_Method(ObjectiveC2_State state, BinaryReader reader,
 
 		if (isSmallList) {
 			int nameOffset = (int)ObjectiveC1_Utilities.readNextIndex(reader, true);
-			int namePtr = reader.readInt(_index + nameOffset);
-			long imagebase = state.program.getImageBase().getOffset(); // When we support dyld_shared_cache, this base will likely have to change
-			name = reader.readAsciiString(imagebase + namePtr);
+			long namePtr;
+			if (state.is32bit)
```

Comment:
you will need to reformat this a bit to fit with our coding standards.  There are code formatter settings for eclipse checked into git that will auto-format most of it.  In this case, you need to put brackets around the statements, even though it is a single statement.

```
if ( test ) {
  statement();
}
else {
  statement();
}
```